web/src/components/ToolbarItem.tsx

```diff
@@ -1,11 +1,34 @@
 import React, { ReactElement } from 'react'
 import styled from 'styled-components'
 
+import dimensions from '../constants/dimensions'
+import CleanAnchor from './CleanAnchor'
 import StyledSmallViewTip from './StyledSmallViewTip'
-import StyledToolbarItem from './StyledToolbarItem'
 import Button from './base/Button'
 import Icon from './base/Icon'
 
+const StyledToolbarButton = styled(Button)`
+  margin: 8px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 4px;
+
+  @media ${dimensions.smallViewport} {
+    line-height: 1.15;
+  }
+`
+
+const StyledToolbarLink = styled(CleanAnchor)`
+  margin: 8px;
+  color: ${props => props.theme.colors.textColor};
+  text-align: center;
+
+  @media ${dimensions.smallViewport} {
+    line-height: 1.15;
+  }
```

Comment:
:question: Why are you applying different styles for buttons and links? For example the gap is missing for links now. Uniform styling and less duplicated code was the reason for the use of the `as` prop below.